actors/power/src/lib.rs

```diff
@@ -396,6 +399,26 @@ impl Actor {
         Ok(MinerRawPowerReturn { raw_byte_power, meets_consensus_minimum })
     }
 
+    /// Returns the total number of miners created, regardless of whether or not
+    /// they have any pledged storage.
+    fn miner_count(rt: &mut impl Runtime) -> Result<MinerCountReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let st: State = rt.state()?;
+
+        Ok(MinerCountReturn { miner_count: st.miner_count })
+    }
+
+    /// Returns the total number of miners that have more than the consensus minimum amount of storage active.
+    /// Active means that the storage must not be faulty.
+    fn miner_consensus_count(
+        rt: &mut impl Runtime,
+    ) -> Result<MinerConsensusCountReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let st: State = rt.state()?;
+
+        Ok(MinerConsensusCountReturn { miner_consensus_count: st.miner_above_min_power_count })
+    }
```

Comment:
I reckon we should just merge these into a single method with compound return value. There's no extra cost/work in one vs the other: they're both vanilla state reads

Reply:
You convinced me otherwise.